### PR TITLE
cybertime.finance

### PR DESCRIPTION
**Pull request by Whiskas1407**
cybertime.finance logo and space

#### Hi! What is your PR about?

- [x] Add images for your space
- [ ] Add custom domain for your space
- [ ] Add alias for migrate your space
- [ ] Add or edit a skin
- ~Edit your space with GitHub?~ (deprecated) Please migrate your space to ENS https://docs.snapshot.page/spaces/migrate
- ~Create your space with GitHub~ (deprecated) Create your space with ENS instead https://docs.snapshot.page/spaces/create

### CODE REVIEW

**Comment by dimabosh** [comment]
How get approve this commit?

File: spaces/domains.json [hunk 1]
```diff
@@ -217,5 +217,6 @@
   "vote.moonpirate.finance": "moonpirate.eth",
   "vote.mai.finance" : "qidao.eth",
   "dao.modefi.io" : "mod.eth",
-  "vote.koaladefi.eth": "koaladefi.eth"
+  "vote.koaladefi.eth": "koaladefi.eth",
+  "cybertime.finance": "cybertime.finance"
```
Comment (gokaiorg) [inline spaces/domains.json]: You will need to use a subdomain and your ENS.
```suggestion
  "vote.cybertime.finance": "cybertime.eth"
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]
Your space name directory should be similar to your ENS. Follow this example `spaces/cybertime.eth/logo.png`.

**Comment by dimabosh** [comment]
What we must fix now?

**Review by gokaiorg — CHANGES_REQUESTED** [review]
Please delete `spaces/cybertime.finance/logo.png` and `spaces/cybertime.finance/space.png` images

**Comment by gokaiorg** [comment]
- Closing this PR for now, feel free to open it again once you've done the changes.
- We have a new method to insert your images without creating a Pull Request. You can now use the Avatar field as your space settings.
image

**Pull request closed without merging**